pages/news/2020-05-16-intro-outro-sections.html

```diff
@@ -15,13 +15,15 @@
 
 When analyzing a track, Mixxx detects where the first and last sounds are to make it easy to skip playing silent parts at the beginning and end of the track.
 The analyzer places the intro start point at the first sound and the outro end point at the last sound. The first and last sound are determined by the first time the signal rises above -60 dBFS and the last time it goes below -60 dBFS.
-<!-- screenshot -->
+
+<img src="/static/images/news/intro-outro/intro-start-outro-end.png" style="max-width: 465px;" alt="A track with the automatically placed intro start and outro end cues"/>
```

Comment:
```suggestion
![A track with the automatically placed intro start and outro end cues](/static/images/news/intro-outro/intro-start-outro-end.png) 
```

Reply:
I think we need CSS for this. It seems to be working with 1e6b3d4 when making my browser window very narrow.

Reply:
I'd rather not mix Markdown and HTML. Can we try to use plain markdown and handle the image resizing in CSS automatically?

This should work. It doesn't stretch small images to the full page width (it centers them instead), but large images are downscaled if they wouldn't fit:

```css
img {
  /* Center the image horizontally */
  display:block;
  margin-left:auto;
  margin-right:auto;
  /* Automatically downscale image to fit in parent container */
  max-width: 100%;
  height: auto;
}
```

If we can, we should use markdown only. That would also make the transition easier if we want to switch away from cactus (e.g. to Hugo or something like that).

Reply:
Check #93